Doc/library/asyncio-stream.rst

```diff
@@ -49,8 +49,8 @@ and work with streams:
 
 
 .. coroutinefunction:: open_connection(host=None, port=None, \*, \
-                          loop=None, limit=None, ssl=None, family=0, \
-                          proto=0, flags=0, sock=None, local_addr=None, \
+                          limit=None, ssl=None, family=0, proto=0, \
```

Comment:
Drop `The *loop* argument is optional...` sentence below and for all four functions.

Reply:
Done